ddtrace/bootstrap/sitecustomize.py

```diff
@@ -98,12 +95,7 @@ def update_patched_modules():
         trace_enabled = True
     else:
         trace_enabled = False
-        opts["enabled"] = False
```

Comment:
Why this change?

Reply:
I can revert, but it isn't needed. We read `DD_TRACE_ENABLED` for tracer init anyways.

https://github.com/DataDog/dd-trace-py/blob/e27f60181/ddtrace/tracer.py#L187-L187

Reply:
We can revert to keep this PR minimal.

Reply:
let's follow up with it!